x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -275,13 +278,21 @@ const getColumns = ({
         }
       };
       return (
-        <EuiSwitch
-          className="eui-textTruncate"
-          checked={!isRuleMuted}
-          onChange={useChangeCspRuleStateFn}
-          data-test-subj={RULES_ROWS_ENABLE_SWITCH_BUTTON}
-          label=""
-        />
+        <EuiFlexGroup justifyContent="flexEnd">
+          <EuiFlexItem grow={false}>
+            <EuiSwitch
+              className="eui-textTruncate"
+              checked={!isRuleMuted}
+              onChange={useChangeCspRuleStateFn}
+              data-test-subj={RULES_ROWS_ENABLE_SWITCH_BUTTON}
+              label=""
+              compressed={true}
+              css={css`
+                padding-left: 18px;
+              `}
```

Comment:
why? can you add a screenshot? i think making the column the same width as the widget will be better